pkg/ghclients/memorycache.go

```diff
@@ -0,0 +1,61 @@
+// Copyright © 2012 Greg Jones (greg.jones@gmail.com)
+// Copyright 2021 Allstar Authors
+
+// Permission is hereby granted, free of charge, to any person obtaining a copy
+// of this software and associated documentation files (the “Software”), to
+// deal in the Software without restriction, including without limitation the
+// rights to use, copy, modify, merge, publish, distribute, sublicense, and/or
+// sell copies of the Software, and to permit persons to whom the Software is
+// furnished to do so, subject to the following conditions:
+
+// The above copyright notice and this permission notice shall be included in
+// all copies or substantial portions of the Software.
+
+// THE SOFTWARE IS PROVIDED “AS IS”, WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+// IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+// FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+// AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+// LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING
+// FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS
+// IN THE SOFTWARE.
+package ghclients
+
+import "sync"
+
+// memoryCache is an implemtation of httpcache.Cache that stores responses in
+// an in-memory map.  It is a copy of httpcache.MemoryCache but exposes Items
+// to allow for calculating and logging the cache size.
+type memoryCache struct {
+	mu    sync.RWMutex
+	Items map[string][]byte
```

Comment:
Nit: `Items` is not exported outside the package. Can be lowercase, i.e `items`

Reply:
It is accessed outside the struct.